app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationConfirmationAPI4.scala.html

```diff
@@ -5,9 +5,10 @@
     <p @pStyle>What happens next</p>
     <p @pStyle>
         <ul>
-            <li>There is no need to contact HMRC, you will be notified of the outcome of your application in due course. You may not receive a decision before 1 April 2017.</li>
-            <li>From 1 April 2017, you must not trade in wholesale alcohol if you do not have an AWRS approval from HMRC.</li>
-            <li>If you continue to trade from 1 April 2017 without an approval you may be liable to financial penalties and your alcohol stock could be confiscated (whether or not duty has been paid).</li>
+            <li>There is no need to contact HMRC, you will be notified of the outcome of your application in due course.</li>
+            <li>As you were trading as an alcohol wholesaler before 31 March 2016, your application should have been made before that date.</li>
```

Comment:
Could you please confirm that 2016 is the intended year? The previous version was 2017.